packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+import { Notification } from '../../graphql/notifications';
+import { useDomPurify } from '../../hooks/useDomPurify';
+import NotificationItemAttachment from './NotificationItemAttachment';
+import NotificationItemAvatar from './NotificationItemAvatar';
+
+export interface NotificationItemProps
+  extends Pick<
+    Notification,
+    'type' | 'icon' | 'title' | 'description' | 'avatars' | 'attachments'
+  > {
+  isUnread?: boolean;
+  targetUrl?: string;
+}
+
+function NotificationItem({
```

Comment:
Where do we handle the `targetUrl`?
Another suggestion is to have the root object to actually be an anchor tag that directs to `targetUrl` and we can intercept the click and change it to open modals when needed